editor/src/core/shared/element-path.ts

```diff
@@ -73,6 +73,28 @@ function getElementPathCache(fullElementPath: ElementPathPart[]): ElementPathCac
 const SceneSeparator = ':'
 const ElementSeparator = '/'
 
+function getComponentPathStringForPathString(path: string): string | null {
+  const indexOfLastSceneSeparator = path.lastIndexOf(SceneSeparator)
+  const indexOfLastElementSeparator = path.lastIndexOf(ElementSeparator)
+  if (indexOfLastSceneSeparator > indexOfLastElementSeparator) {
```

Comment:
It is really implicit here what happens if one (or both) of the indexes is/are -1
I would handle this even if it doesn't happen IRL

Reply:
This is actually by design, but I have no issue making it explicit :+1: